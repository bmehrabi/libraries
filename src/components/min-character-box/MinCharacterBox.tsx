import Box from '@mui/material/Box';
import * as React from 'react';
import {IMinCharacterBoxProps} from './typings';
import Title from 'src/components/Title';

export default function MinCharacterBox ({minLength}: IMinCharacterBoxProps) {
  return (
    <Box sx={{ m: 2 }}>
      <Title>
        Please enter a search query inside search field (at least {minLength} letters).
      </Title>
    </Box>
  );
}