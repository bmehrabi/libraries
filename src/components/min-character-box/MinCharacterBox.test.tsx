import MinCharacterBox from './MinCharacterBox';
import {render} from '@testing-library/react';

describe('MinCharacterBox Component', () => {
  it('matches the snapshot', () => {
    const {container} = render(<MinCharacterBox minLength={3} />);

    expect(container).toMatchSnapshot();
  });
})