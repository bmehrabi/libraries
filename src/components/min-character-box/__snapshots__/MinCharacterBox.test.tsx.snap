// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinCharacterBox Component matches the snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-1c5ij41"
  >
    <h2
      class="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom css-gi7hi0-MuiTypography-root"
    >
      Please enter a search query inside search field (at least 
      3
       letters).
    </h2>
  </div>
</div>
`;
